app/views/partials/foyer/individu-form.html

```diff
@@ -212,6 +212,16 @@
     <question>{{ isDemandeur() ? 'Êtes-vous' : 'Est-il/elle' }} étudiant boursier&nbsp;?</question>
   </yes-no-question>
 
+  <yes-no-question model="individu.aCharge"
+    ng-if="captureDemandeurAChargeParents()"
+    size="4"
+    >
+    <question>Êtes-vous à charge fiscale de vos parents&nbsp;?</question>
+    <help-block>
+      Répondez "Non" si vous avez déjà fait une déclaration de revenus en votre nom, ou si vous allez le faire pour l'année à venir.
```

Comment:
> « Qui déclare vos revenus cette année ? » me laisse dubitatif.

Ok, mais « Êtes-vous à charge fiscale de vos parents ? » me semble une question dont la réponse est très peu évidente. On peut itérer ?  :wink:

> Que se passe-t-il si je n'ai pas de revenu ?

Tu dois quand même faire une déclaration aux impôts.

> Est-ce évident qu'on parle d'une déclaration fiscale ?

Non. « Qui déclare vos revenus aux impôts cette année ? » ?